azurerm/resource_arm_database_vulnerability_assessment_rule_baseline_test.go

```diff
@@ -0,0 +1,277 @@
+	resourceName := "azurerm_database_vulnerability_assessment_rule_baseline.test"
+	ri := tf.AccRandTimeInt()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMStorageAccountDatabaseDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMDatabaseVulnerabilityAssessmentRuleBaseline_master(ri, testLocation()),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMBaselineExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "rule_id", "VA2065"),
+					resource.TestCheckResourceAttr(resourceName, "baseline_name", "master"),
+					resource.TestCheckResourceAttr(resourceName, "baseline_result.#", "1"),
+				),
+			},
+			{
+				Config: testAccAzureRMDatabaseVulnerabilityAssessmentRuleBaseline_update(ri, testLocation()),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMBaselineExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "rule_id", "VA2065"),
+					resource.TestCheckResourceAttr(resourceName, "baseline_name", "master"),
+					resource.TestCheckResourceAttr(resourceName, "baseline_result.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func testCheckAzureRMBaselineExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("resource not found: %s", resourceName)
+		}
+
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+		serverName := rs.Primary.Attributes["server_name"]
+		databaseName := rs.Primary.Attributes["database_name"]
+		ruleId := rs.Primary.Attributes["rule_id"]
+		baselineName := expandBaselineName(rs.Primary.Attributes["baseline_name"])
+
+		client := testAccProvider.Meta().(*ArmClient).databaseVulnerabilityAssessmentRuleBaselinesClient
+		ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+		resp, err := client.Get(ctx, resourceGroup, serverName, databaseName, ruleId, *baselineName)
+		if err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("baseline was not found for "+
+					"resource group %q, sql server %q, database %q, rule id %q, baseline name %q",
+					resourceGroup, serverName, databaseName, ruleId, *baselineName)
+			}
+
+			return err
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMStorageAccountDatabaseDestroy(s *terraform.State) error {
+	err := testCheckAzureRMStorageAccountDestroy(s)
+
+	if err != nil {
+		return err
+	}
+
+	return testCheckAzureRMSqlDatabaseDestroy(s)
+}
+
+func testAccAzureRMDatabaseVulnerabilityAssessmentRuleBaseline_basic(rInt int, location string) string {
+
+	database := testAccAzureRMDatabaseVulnerabilityAssessmentRuleBaseline_database(rInt, location)
+
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_database_vulnerability_assessment_rule_baseline" "test" {
+  resource_group_name = "${azurerm_resource_group.test.name}"
```

Comment:
Could we format this TF and align the assignments?

Reply:
done